### PR TITLE
update wxwidgets, from xenial we need to use gtk3.0

### PR DESCRIPTION
**Pull request by k-okada**
update wxwidgets, from xenial we need to use gtk3.0

I decied to switch to gtk 3.0 from xenial, as wxpython entry do
http://packages.ubuntu.com/trusty/libwxgtk3.0-dev

### CODE REVIEW

**Review by dirk-thomas — APPROVED** [review]
This will affect already released packages depending in this key. They need to be rebloomed to get the new mapping.

**Comment by k-okada** [comment]
yes, and these packages should be failed within the build firm.
http://build.ros.org/job/Kbin_uX64__jsk_rviz_plugins__ubuntu_xenial_amd64__binary/3/console#console-section-12


--
◉ Kei Okada


2017-02-14 10:55 GMT+09:00 Dirk Thomas <notifications@github.com>:

File: rosdep/base.yaml [hunk 1]
```diff
@@ -4160,7 +4160,23 @@ wxwidgets:
   macports: [wxWidgets-python]
   opensuse: [wxGTK-devel]
   rhel: [wxGTK-devel]
-  ubuntu: [libwxgtk2.8-dev]
+  ubuntu:
```
Comment (tfoote) [inline rosdep/base.yaml]: This line is is here twice.

**Review by tfoote — CHANGES_REQUESTED** [review]
This should also be updated for debian: It lookslike 3.0 was published as of Jessie: https://packages.debian.org/jessie/libwxgtk3.0-dev
And 2.8 was last in wheezy: https://packages.debian.org/wheezy/libwxgtk2.8-dev

http://packages.ubuntu.com/trusty/libwxbase2.8-dev vs http://packages.ubuntu.com/trusty/libwxgtk3.0-dev

**Comment by wjwwood** [comment]
Also, perhaps a bit off-topic, but why does your `jsk_rviz_plugins` package depend on `wx`?

File: rosdep/base.yaml (hunk 1 above)
Comment (k-okada) [inline rosdep/base.yaml]: @tfoote sorry, fixed now.

**Comment by k-okada** [comment]
> Also, perhaps a bit off-topic, but why does your jsk_rviz_plugins package depend on wx?

@wjwwood Good question, may be we do not need that. I just afraid something bad happens by changing dependency. I'll try to remove that https://github.com/jsk-ros-pkg/jsk_visualization/pull/664
https://github.com/jsk-ros-pkg/jsk_visualization/commit/a7f3f76f86efcc73069eb1f840ccab90f5c12e00

**Comment by wjwwood** [comment]
I mean, do whatever works for you. I don't mind that you're using `wx`, and I think we should fix the rosdep rule. Just curious.

File: rosdep/base.yaml (hunk 1 above)
Comment (tfoote) [inline rosdep/base.yaml]: +1

File: rosdep/base.yaml [hunk 2]
```diff
@@ -4153,14 +4153,32 @@ wxpython:
     xenial: [python-wxgtk3.0]
 wxwidgets:
   arch: [wxgtk]
-  debian: [libwxgtk2.8-dev]
+  debian:
+    jessie: [libwxgtk3.0-dev]
+    stretch: [libwxgtk3.0-dev]
```
Comment (tfoote) [inline rosdep/base.yaml]: Thanks for the debian updates. I removed the sid entry since we don't target sid because it's a rolling target.

**Review by tfoote — APPROVED** [review]

**Pull request merged**